modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,26 +78,44 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
 
+|`Import`    |Import configuration options for Argo CD.| `__<object>__`
+a|* `<Name>` - The name of an `ArgoCDExport` resource from which data can be imported.
+  * `<Namespace>` - The namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.
```

Comment:
From:
- The namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.

To:
- Namespace for the `ArgoCDExport` referenced by `name` field, above. If this field is not set, the namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.

Reply:
Fixed.